src/ArrayValueCalculator.java

```diff
@@ -0,0 +1,30 @@
+import exception.ArrayDataException;
+import exception.ArraySizeException;
+
+public class ArrayValueCalculator {
+    private static final int ARRAY_ROW_NUMBER = 4;
+    private static final int ARRAY_COLUMN_NUMBER = 4;
+
+    public int doCalc(String[][] array) throws ArraySizeException, ArrayDataException {
+        int result = 0;
```

Comment:
Would be better to declare this variable right after the validation, because it does not have any sense to allocate memory before validation is passed.